index.html

```diff
@@ -0,0 +1,154 @@
+          <h2 class="Home-1-h2">"Hello! Playstation world"</h2>
+          <h1>
+            PLAYSTATION & Insomniac Games<br />
+            GAMES SHOWCASE<br />
+            2022
+          </h1>
+          <p>
+            Play brand new games the same day they release and hundreds of
+            additional high-quality games with Playstation Game Pass. Join us as
+            we showcase even more additions to your game pass catalog.
+          </p>
+          <h2>2021.10.27(TUE) ~ 28(WED)</h2>
+          <p>@ Redmond, Washington, US, Microsoft Headquarters</p>
+        </div>
+      </section>
+    </div>
+    <section class="main-page" id="Main-Program">
+      <h1 class="main-page-1">Main Program</h1>
+      <div class="border"></div>
+      <!-- Card 1 starting -->
+      <div class="card margin-l">
+        <div class="card-1 left">
+          <img src="./Img/Card-Icon/launch.png" alt="icon of rocket" />
+          <h5>launch</h5>
+          <p>
+            Show opener introductions with Erin Ashely and Malik. Secial look at
+            fans celebration of the xbox games showcase
+          </p>
+        </div>
+        <!-- Card -1 end -->
+        <!-- Card 2 starting -->
+        <div class="card-1 left card-im">
+          <img
+            src="./Img/Card-Icon/playstation-png.png"
+            alt="icon of Playstation"
+          />
+          <h5>Playstation</h5>
+          <p>
+            Console exclusive games showcase. This includes PS5 new version
+            console exclusive trailer.
+          </p>
+        </div>
+        <!-- Card 2 end -->
+        <!-- Card 3 starting -->
+        <div class="card-1 left card-im">
+          <img
+            src="./Img/Card-Icon/ps-controler.png"
+            alt="icon of Playstation controler"
+          />
+          <h5>Controler</h5>
+          <p>
+            Console exclusive games showcase. This includes new Controler 
+            exclusive trailer.
+          </p>
+        </div>
+        <!-- Card 3 end -->
+        <!-- Card 4 starting -->
+        <div class="card-1 left">
+          <img src="./Img/Card-Icon/behind-the-scenes.png" alt="icon of Behind the scenes" />
+          <h5>Behind the scenes</h5>
+          <p>
+            Behind the scenes:-  Create a game like new relase game Spiderman 
+            next year relaese game god of war. 
+          </p>
+        </div>
+        <!-- Card 4 end -->
+        <!-- Card 5 starting -->
+        <div class="card-1">
+          <img src="./Img/Card-Icon/peakpx.jpg" alt="icon of rocket" />
+          <h5>launch Future</h5>
+          <p>
+            Showcase the game of GOD OF WAY RAGNAROK. The new exlusive the 
+            game of Playstation.
+          </p>
+        </div>
+        <!-- Card 5 end -->
+      </div>
+      <button><a href="https://www.youtube.com/watch?v=T5VtfD2i3rc">Watch Playstation Showcase</a></button>
+    </section>
+    <section class="Featured" id="Featured"></section>
```

Comment:
- [ ] Good job making the changes to your featured section, But as requested it is still not matching the design guidelines, please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/203846970-1337342f-2a68-4098-94cb-08115853d661.png)

I think we can improve this by setting 
```
.maincardsec {
  display: flex;
  width: 100%;
  justify-content: center;
  flex-direction: row;
  gap: 24px;
}
```